understanding/bypass-blocks.html

```diff
@@ -55,109 +53,82 @@
-                  repeated content on each page. This is in contrast to a sighted user's ability to
-                  ignore
-                  the repeated material either by focusing on the center of the screen (where main content
-                  usually appears) or a mouse user's ability to select a link with a single mouse click
-                  rather than encountering every link or form control that comes before the item they
-                  want.
+               <p>コンテンツを順を追ってナビゲートする利用者は通常、各ページで繰り返されるコンテンツを都度ナビゲートしなければならない。これは、目の見える利用者が画面の中央 (通常はメインコンテンツが現れるところ) に注目して繰り返される要素を無視したり、マウスクリック 1 回でリンクを選択したりすることによって、選択したい要素より前にあるリンクもしくはフォームコントロール全てに出会わずに済ませる能力があることとは、対照的なものである。
                   
                </p>
-               <p>It is not the intent of this Success Criterion to require authors to provide methods
-                  that are redundant to functionality provided by the user agent. Most web browsers
-                  provide keyboard shortcuts to move the user focus to the top of the page, so if a
-                  set of navigation links is provided at the bottom of a web page providing a "skip"
-                  link may be unnecessary.
+               <p>この達成基準の意図は、コンテンツ制作者にユーザエージェントが提供している機能と重複する手段を提供するように求めることではない。ほとんどのウェブブラウザは、利用者がフォーカスをページの先頭に移動するためのキーボードショートカットを提供しているので、ナビゲーションリンク一式をウェブページの下部で提供している場合は、「スキップ」リンクの提供は不要かもしれない。
                   
                </p>
                <div class="note">
-                  <p class="note-title marker">Note</p>
+                  <p class="note-title marker">注記</p>
                   <div>
                      
                      
-                     <p>Although this Success Criterion deals with blocks of content that are repeated on
-                        multiple pages, we also strongly promote structural markup on individual pages as
-                        per Success Criteria 1.3.1.
-                        
+                     <p>この達成基準は、複数のページ上で繰り返されているコンテンツのブロックを扱っているが、単独のページにおいて、達成基準 1.3.1 にあるように、構造をマークアップすることも強く推奨する。
                         
                      </p>
                      
                      
                   </div>
                </div>
-               <p>Although the success criterion does not specifically use the term “within a set of
-                  web pages”, the concept of the pages belonging to a set is implied.  An author would
-                  not be expected to avoid any possible duplication of content in any two pages that
-                  are not in some way related to each other, and are not "Web pages that share a common
-                  purpose and that are created by the same author, group or organization” (the definition
-                  of set of web pages).
+               <p>この達成基準では、「ウェブページ一式において」という用語は使用していないものの、ページが集合に属するという概念は示唆されている。コンテンツ制作者は、相互になんらかの関係がなく、かつ「共通の目的を共有し、同じコンテンツ制作者、グループ、又は組織により制作されたウェブページの集合」(ウェブページ一式の定義) ではない二つのページにおいて、コンテンツが重複する可能性を避けることが期待されるということはないだろう。
```

Comment:
これでよいのかどうかが謎ですが…

> the concept of the pages belonging to a set is implied.

「a set」について、後続の注記にあわせて「一つの集合」としてみました
「the pages」について、`「…ウェブページの集合」(ウェブページ一式の定義)`から、「ページの集合」としてみました（複数のページであることを明示的に）

また、文末がやや冗長になっているのを修正してみました

```suggestion
               <p>この達成基準では、「ウェブページ一式において」という用語は使用していないものの、一つの集合に属するページの集合の概念が示唆されている。コンテンツ制作者は、相互になんらかの関係がなく、かつ「共通の目的を共有し、同じコンテンツ制作者、グループ、又は組織により制作されたウェブページの集合」(ウェブページ一式の定義) ではない二つのページにおいて、コンテンツが重複する可能性を避けることは期待されないだろう。
```